bluemira/builders/cryostat.py

```diff
@@ -146,8 +154,8 @@ def build_xz(self, x_out: float, z_top: float) -> PhysicalComponent:
         x_outer[1:-1] += tk
 
         z_outer = np.array([z_bot, z_bot, z_mid, z_mid, z_top, z_top])
-        z_outer[:2] -= tk
-        z_outer[2:] += tk
+        z_outer[:4] -= tk
+        z_outer[4:] += tk
```

Comment:
Can confirm: this is right, but I have a feeling we've either changed this recently, or changed it the same way somewhere else. Anyway, let's not touch it any more!

Reply:
It was probably me in the conversion...at least its right now